snapm/manager/_manager.py

```
@@ -580,10 +783,19 @@
 
     @suspend_signals
     def __init__(self):
+        # Verify existence of lock path before initialising Manager
+        try:
+            self._lock_dir = _check_lock_dir()
+        except OSError as err:
+            raise SnapmNotFoundError(
+                f"Failed to create lock directory at {_SNAPSET_LOCK_DIR}: {err}"
+            ) from err
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)